TUTORIAL.md

```diff
@@ -558,11 +567,11 @@ export class HomeComponent {
 }
 ```
 
-You should be able to sign in using the form, using one of your app's registered users. After logging in, you'll be able to click the **Beer List** link and view the beer list.
+You should be able to sign in with the form as one of your app's registered users. After logging in, you'll be able to click the **Beer List** link and view the beer list.
 
-## Authenticate with Stormpath Angular SDK
+## Authenticate with the Stormpath Angular SDK
 
-If you're a former Stormpath migrating to Okta, you might've used our Angular, AngularJS, or React SDK. The good news is these libraries should still work with Okta! 
+If you're a former Stormpath customer migrating to Okta, you might've used our Angular, AngularJS, or React SDKs. The good news is that these libraries should still work with Okta! ** -- what if they weren't a stormpath user? there's no way to register for stormpath anymore... can they still download and use the sdk? i assume they can... either way, we should add a line about new okta users trying this out, andk how they can expect it to be supported going forward.**
 
```

Comment:
This is the hard part of a PR. If it's coming from your repo, I don't have access to make changes unless you add me as a contributor to your forked repo. If the branch is created on the `oktadeveloper` repo, I can checkout the branch and make changes.